routers/api/v1/repo/pull.go

```diff
@@ -8,6 +8,8 @@ import (
 	"fmt"
 	"strings"
 
+	"code.gitea.io/gitea/modules/notification"
+
```

Comment:
Why is this in a separate import paragraph?

Reply:
done.